### ISSUE TITLE
error: use of undeclared identifier 'c_void'

### ISSUE THREAD

**Issue by williamstein** [ziglang/zig#10391]
### Zig Version

0.9.0

### Steps to Reproduce

Use c_void anywhere in a zig program with zig the newly release 0.9.0, and you'll get
```
error: use of undeclared identifier 'c_void'
```
However, the docs for 0.9.0 (https://ziglang.org/documentation/0.9.0) still list c_void in numerous places.  Evidently it was deprecated.

Searching the zig standard library source code, I discovered that using `anyopaque` instead of `c_void` seems to work as a replacement.

### Expected Behavior

Either c_void works as before (up to a few days ago), or c_void is not in the official docs, and instead the docs suggest using `anyopaque`.

### Actual Behavior

```
error: use of undeclared identifier 'c_void'
```

**Comment by williamstein** [comment]
Note that the docs for master have c_void deleted, so fixing this may just be a matter of slightly updating the 0.9.0 docs, which maybe got out of sync when doing the release?

**Comment by andrewrk** [comment]
The bug tracker is for master branch, which does not have this issue. Apologies for the 0.9.0 docs having the bug. It will be fixed in 0.9.1. Here's the relevant release notes section: https://ziglang.org/download/0.9.0/release-notes.html#c_void-renamed-to-anyopaque

If you run `zig fmt` on your codebase it will fix this issue for you.
